clang/test/CIR/CodeGen/gnu-extendion.c

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
+
+int foo(void) { return __extension__ 0b101010; }
+
+//CHECK: cir.func @foo() -> !s32i extra( {inline = #cir.inline<no>, optnone = #cir.optnone} ) {
+//CHECK-NEXT:    %0 = cir.alloca !s32i, cir.ptr <!s32i>, ["__retval"] {alignment = 4 : i64}
```

Comment:
Minor nitpicks here:
- The file name is misspelled, should be `gnu-extension`. 
- Use FileCheck regexes for MLIR values, for example:
```
%[[addr:.*]] = cir.alloca !s32i, cir.ptr <!s32i>,
%[[const:.*]] = cir.const(#cir.int<42> : !s32i) : !s32i
cir.store %[[const]], %[[addr]] : !s32i, cir.ptr <!s32i>
```

Reply:
Ok, got it

Reply:
Fixed